frontend/src/lib/components/SchemaEditor.svelte

```diff
@@ -204,55 +250,15 @@
 							<th />
 						</tr>
 						<tbody slot="body">
-							{#each Object.entries(schema.properties) as [name, property], i (name)}
-								<tr animate:flip>
-									<td class="font-bold">{name}</td>
-									<td>
-										<SchemaEditorProperty {property} />
-									</td>
-									<td>{property.description ?? ''}</td>
-									<td>{property.default ? JSON.stringify(property.default) : ''}</td>
-									<td
-										>{#if schema.required.includes(name)}
-											<span class="text-red-600 font-bold text-lg">*</span>
-										{/if}</td
-									>
-									<td class="justify-end flex">
-										{#if i > 0}
-											<button
-												on:click={() => changePosition(i, true)}
-												class="text-lg mr-2 {isAnimated ? 'invisible' : ''}"
-											>
-												&uparrow;</button
-											>
-										{/if}
-										{#if i < Object.keys(schema.properties).length - 1}
-											<button
-												on:click={() => changePosition(i, false)}
-												class="text-lg mr-2 {isAnimated ? 'invisible' : ''}">&downarrow;</button
-											>
-										{/if}
-
-										<Button
-											color="red"
-											variant="border"
-											btnClasses="mx-2"
-											size="sm"
-											startIcon={{ icon: faTrash }}
-											on:click={() => handleDeleteArgument(name)}
-										>
-											Delete
-										</Button>
-										<Button
-											color="light"
-											variant="border"
-											size="sm"
-											startIcon={{ icon: faPen }}
-											on:click={() => startEditArgument(name)}
-										>
-											Edit
-										</Button>
-									</td>
+							{#each schemaPropertiesToDisplay(schema) as displayInfo (displayInfo.name)}
```

Comment:
`
export async function main(x: string, a: {x: number, y: {x:boolean}}) {
  return a.y.x
}
`
This code generates a valid schema, but property names are clashing and makes the app to crash
I think we might just use the index